content/docs/howto/java.md

```diff
@@ -407,24 +408,50 @@ Connect your IDE debugger to connect to the published port.
 
 ## Enable JMX
 
-If `BP_JMX_ENABLED` is set at build-time and `BPL_JMX_ENABLED` is set at runtime, the [JMX Buildpack][bp/jmx] will enable [JMX][jmx]. The JMX connector will listen on port `5000` by default. The port can be configured with the `BPL_JMX_PORT` environment variable at runtime.
+If `BPL_JMX_ENABLED` is set at runtime, the application will be configured to accept [JMX][jmx] connections. The JMX connector will listen on port `5000` by default. The port can be configured with the `BPL_JMX_PORT` environment variable at runtime.
 
 **Example**: Enabling JMX
 
-The following commands builds a JMX enabled image.
-{{< code/copyable >}}
-pack build samples/java \
-  --path java/jar \
-  --env BP_JMX_ENABLED=true
-{{< /code/copyable >}}
-
-To run the image with the JMX port published:
+To run an image with the JMX port published:
 {{< code/copyable >}}
 docker run --env BPL_JMX_ENABLED=true --publish 5000:5000 samples/java
 {{< /code/copyable >}}
 
 Connect [JConsole][jconsole] to the published port.
 ![JConsole](/images/jconsole.png)
+  
+## Enable Java Native Memory Tracking (NMT)
+
+By default, the JVM will be configured track internal memory usage. The JVM will print its last memory usage data when it exits, the level of detail can be configured at runtime by setting the environment variable `BPL_JAVA_NMT_LEVEL`, which supports both `summary` (default) and `detail`. Since there is a small amount of overhead required to support NMT, it can be disabled by setting the environment variable `BPL_JAVA_NMT_ENABLED` to `false`.
```

Comment:
```suggestion
By default, the JVM will be configured to track internal memory usage. The JVM will print its last memory usage data when it exits, the level of detail can be configured at runtime by setting the environment variable `BPL_JAVA_NMT_LEVEL`, which supports both `summary` (default) and `detail`. Since there is a small amount of overhead required to support NMT, it can be disabled by setting the environment variable `BPL_JAVA_NMT_ENABLED` to `false`.
```